recipes/pcmdi_metrics/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "pcmdi_metrics" %}
+{% set version = "2.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/PCMDI/pcmdi_metrics/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: a2ed18d379d128cc994d78dabb5bc1da39f3331e26fa8a314705a7cc5fee371f
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+  entry_points:
+    - compositeDiurnalStatistics.py = pcmdi_metrics.diurnal.scripts.compositeDiurnalStatistics:main
+    - computeStdOfDailyMeans.py = pcmdi_metrics.diurnal.scripts.computeStdOfDailyMeans:main
+    - fourierDiurnalAllGrid.py = pcmdi_metrics.diurnal.scripts.fourierDiurnalAllGrid:main
+    - fourierDiurnalGridpoints.py = pcmdi_metrics.diurnal.scripts.fourierDiurnalGridpoints:main
+    - savg_fourier.py = pcmdi_metrics.diurnal.scripts.savg_fourier:main
+    - std_of_dailymeans.py = pcmdi_metrics.diurnal.scripts.std_of_dailymeans:main
+    - std_of_hourlyvalues.py = pcmdi_metrics.diurnal.scripts.std_of_hourlyvalues:main
+    - std_of_meandiurnalcycle.py = pcmdi_metrics.diurnal.scripts.std_of_meandiurnalcycle:main
+
+requirements:
+  build:
+    - python >=3.6
+    - setuptools
+    - numpy =1.21.3
+  host:
+    - python >=3.6
+    - pip =21.2.4
+  run:
+    - python >=3.6
+    - cdms2 =3.1.5
+    - genutil =8.2.1
+    - cdutil =8.2.1
+    - cdp =1.7.0
+    - matplotlib =3.4.3
+    - numpy =1.21.3
+    - cartopy =0.20.1
+    - eofs =1.4.0
+    - jupyterlab =3.2.1
+    - nb_conda =2.2.1
+    - nb_conda_kernels =2.3.1
```

Comment:
Is it really necessary to pin all these versions exactly?

Also, unless you need a pyqt-based GUI, better to depend on `matplotlib-base` rather than `matplotlib.

Reply:
@jasonb5 do you have any advice? I copied these from our dev environment. It seems like the python and cdat package versions would be important, but I don't know about the other packages.

Reply:
@dopplershift I removed all the versions except Python, since we don't support py<3.6.